docker/Dockerfile_passenger_max

```diff
@@ -13,6 +13,9 @@ RUN bundle update
 # So we may use whatever command we want to trigger rake, to be sure
 # that the agent is not braking the rake task
 
+# Tesing 
```

Comment:
When we're done, will we still need this? This comes from trying to use both linux distros on one system, right?

Reply:
We would need them yes for the FFI and for loading the dynamic lib, as we use debian for each server image we use to test. Since if not installed there will be dependency error for ffi and AgentLib not able to be loaded due to lack of dependency.